src/write.rs

```diff
@@ -635,10 +744,9 @@ mod maybe_seek {
                 //
                 // See discussion here:
                 //   https://users.rust-lang.org/t/a-trait-object-with-an-implied-lifetime/29340
-                std::mem::transmute::<
-                    Box<dyn WriteSeek<W> + '_>,
-                    Box<dyn WriteSeek<W> + 'static>,
-                >(Box::new(w))
+                std::mem::transmute::<Box<dyn WriteSeek<W> + '_>, Box<dyn WriteSeek<W> + 'static>>(
```

Comment:
gaaaaaaaaahhhhhhh, this really hurts the readibility.

Hmmm maybe we can force rustfmt to leave it alone with a line comment:

```rust
                std::mem::transmute::<
                    Box<dyn WriteSeek<W> + '_>, // this lifetime satisfies W: '_
                    Box<dyn WriteSeek<W> + 'static>,
                >(Box::new(w))
```